### PR TITLE
Add Meter Identification cluster

### PR DESCRIPTION
**Pull request by Michail-Antropov**
In progress

### CODE REVIEW

File: data_model/clusters/MeterIdentification.xml [hunk 1]
```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0"?>
```
Comment (andy31415) [inline data_model/clusters/MeterIdentification.xml:1]: How was this added? data_model files are managed by a script and should be included only as part of script execution (i.e. from parsing asciidoc files)

Comment (Michail-Antropov) [reply data_model/clusters/MeterIdentification.xml:1]: In the image and likeness of other clusters

File: src/app/clusters/meter-identification-server/meter-identification-server.cpp
Comment (bzbarsky-apple) [inline src/app/clusters/meter-identification-server/meter-identification-server.cpp:1]: Please split this into separate PRs for:

1. Add the cluster XML (needs careful review by a set of people)
2. Add the cluster implementation (needs careful review by a slightly different set of people)
3. Add the example app bits.

File: src/app/zap-templates/zcl/data-model/chip/meter-identification-cluster.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/meter-identification-cluster.xml:3]: This does not look like the right year.

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]

File: src/app/zap-templates/zcl/data-model/chip/meter-identification-cluster.xml
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/meter-identification-cluster.xml:1]: I don't see this cluster in the spec, by the way, which makes reviewing this impossible.

**Comment by Michail-Antropov** [comment]
Split to 3 PR
